tests/functional/mesh/netceptor_test.go

```diff
@@ -0,0 +1,127 @@
+package mesh
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/ansible/receptor/tests/functional/lib/mesh"
+)
+
+func TestSetMaxConnectionIdleTimeFromPseudoConfigFile(t *testing.T) {
```

Comment:
are these tests ensuring that the mesh fails to start because of a bad `maxidleconnectiontimeout` setting?

I'd be content with just having the 3 single node tests that you have above, since they basically test the same thing but are less involved.